index/index.go

```diff
@@ -220,6 +221,53 @@ func (idx *Indexer) loopIteration(
 				doc:       doc,
 			}
 		case "document", "acl", "status":
+			query, err := json.Marshal(createLanguageQuery(item.Uuid, language))
+			if err != nil {
+				return 0, fmt.Errorf("marshal json: %w", err)
+			}
+
+			rootAlias := idx.getIndexTypeRoot(item.Type, "documents")
+
+			// Find all existing documents with the same Id but a different
+			// language
+			oldDocsResponse, err := idx.client.Search(
```

Comment:
Started worrying about eventual consistency here, but we wouldn't see inconsistencies until we get high frequency writes that **also** change the language, so :shrug: this is good enough.

Reply:
Consistency in ES/OS is iffy. There are no atomic operations, and by default there is a 1s delay between a write and when that document is visible for read operations. This can be skipped, at a performance penalty.

Reply:
Yeah, no I think this is good enough. Doc language changes are edge cases in themselves, so we should be good.